dotCMS/src/integration-test/java/com/dotcms/rendering/velocity/services/HTMLPageAssetRenderedTest.java

```diff
@@ -79,9 +86,88 @@
     private static Persona persona;
     private static Visitor visitor;
 
+    @DataProvider
+    public static Object[][] cases() throws Exception {
+        if (systemUser == null) {
+            prepareGlobalData();
+        }
+
+        final Container container = createContainer();
+        final Template templateContainer = createTemplate(container);
+
+        final Container fileContainer = createFileContainer();
+        final Template templateFileContainer = createTemplate(fileContainer);
+
+        return new Object[][] {
+                { container, templateContainer },
+                { fileContainer, templateFileContainer}
+        };
+    }
+
+    private static Template createTemplate(final Container container) throws DotSecurityException, WebAssetException, DotDataException {
+        final Template template = new TemplateDataGen().title("PageContextBuilderTemplate"+System.currentTimeMillis())
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The String literal "PageContextBuilderTemplate" appears 4 times in this file; the first occurrence is on line 108](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5067852)